### PR TITLE
Breaks up Buildpacks into subteams

### PR DESCRIPTION
**Pull request by ryanmoran**
The Buildpacks team has a lot of repositories that it maintains. In order to scale with that set of responsibilities and maintain a semblance of ownership, the buildpacks team has been operating as a number of subteams. This PR aims to codify those teams into our tooling so that we can make it a bit easier for team members to track issues/PRs that they nominally own.

### CODE REVIEW

**Comment by Gerg** [comment]
If the Buildpacks area has this much internal complexity, might it make sense to spin it out as its own Working Group (+Paketo?).

cc @cloudfoundry/toc

**Comment by robdimsdale** [comment]
I'd like to request we block this on #564, as I'd like to add myself to the appropriate sub teams in this PR once I'm a member of the current team.

**Comment by robdimsdale** [comment]
Ok, now that I'm a member of the app interfaces team, I've added myself to this PR in the appropriate places and I'd like to flag it as "unblocked"

That being said, it seems like there's a separate thread about spinning out this (potentially including the paketo teams) into a separate working group. @Gerg are you looking for feedback from the buildpacks team on this idea?

**Comment by beyhan** [comment]
This has been discussed during the TOC Meeting on 2023-03-21. From the [TOC meeting notes](https://docs.google.com/document/d/1ng8rAcnrCbTm5RVNAKZ_ptqXiwcBIPgC8rbYp6A8lUo/edit#):
>TOC + Greg conclude to merge PR for now and open a wider issue to think about structure of buildpack groups in the future.

Is this pr ready for review? In parallel we can open an issue to discuss about possibilities for future buidlpack groups setup.

**Comment by robdimsdale** [comment]
> Is this pr ready for review? In parallel we can open an issue to discuss about possibilities for future buidlpack groups setup.

Yes, it should be good to go, thanks

File: toc/working-groups/app-runtime-interfaces.md
Comment (Gerg) [inline toc/working-groups/app-runtime-interfaces.md:111]: Anthony should remain a reviewer. Any promotion to approver should be in a separate PR.

File: toc/working-groups/app-runtime-interfaces.md
Comment (Gerg) [inline toc/working-groups/app-runtime-interfaces.md:259]: Anthony should remain a reviewer.

**Review by Gerg — CHANGES_REQUESTED** [review]

File: toc/working-groups/app-runtime-interfaces.md
Comment (robdimsdale) [reply toc/working-groups/app-runtime-interfaces.md:112]: Sorry, that was a mistake due to Anthony being added after this PR was created.

**Comment by robdimsdale** [comment]
@Gerg fixed - thanks for catching that. This PR should be good to go now.

File: toc/working-groups/app-runtime-interfaces.md
Comment (rkoster) [inline toc/working-groups/app-runtime-interfaces.md:70]: Wouldn't `Buildpacks Docs` be a more descriptive name?

File: toc/working-groups/app-runtime-interfaces.md
Comment (rkoster) [inline toc/working-groups/app-runtime-interfaces.md:82]: Feels like `Buildpacks Utilities`, `Buildpacks Tooling`, and `Buildpacks Dependencies` could maybe be merged. Working group areas with a single repo that has a generic name do not make sense to me.

File: toc/working-groups/app-runtime-interfaces.md
Comment (robdimsdale) [reply toc/working-groups/app-runtime-interfaces.md:70]: We based these teams off the existing [Paketo teams](https://github.com/orgs/paketo-buildpacks/teams). Happy to change them if we want; we just opted for the same names for consistency and to reduce confusion.

File: toc/working-groups/app-runtime-interfaces.md
Comment (robdimsdale) [reply toc/working-groups/app-runtime-interfaces.md:82]: Same as above - happy to make changes but we started with the Paketo teams for consistency.

**Comment by robdimsdale** [comment]
@rkoster as mentioned inline, we based these teams off the existing [Paketo teams](https://github.com/orgs/paketo-buildpacks/teams). Happy to change them if we want; we just opted for the same names for consistency and to reduce confusion.

How strongly do you feel about your proposal vs mirroring the existing paketo teams?

File: toc/working-groups/app-runtime-interfaces.md
Comment (robdimsdale) [reply toc/working-groups/app-runtime-interfaces.md:70]: Also, to provide more context on the naming of the equivalent Paketo group - `Content` was chosen (rather than docs) because on the Paketo side that sub group is responsible for additional content beyond documentation - they maintain the website and blog as well.

Given we don't have that equivalent for CF buildpacks, I'm happy to call it `Docs` if you prefer.

File: toc/working-groups/app-runtime-interfaces.md
Comment (robdimsdale) [reply toc/working-groups/app-runtime-interfaces.md:82]: On further reflection, I'm happy to merge the `Dependencies` team into `Tooling`. I would prefer to keep `Utilities` and `Tooling` separate, because they are fundamentally different concerns and have distinct areas of expertise.

`Utilities` is a set of buildpacks that don't fit into any particular language family (like the `apt` buildpack, or `Procfile` on the Paketo side).

`Tooling` is the infrastructure and tooling to support the creation and maintenance of Buildpacks (like CI pipelines, test libraries, etc).

**Comment by robdimsdale** [comment]
@rkoster you essentially requested three changes:
1. Rename `Content` to `Docs`
2. Merge `Dependencies` into `Tooling`
3. Merge `Utilities` into `Tooling`.

I'm happy with 1. and 2. but I'd like to reject 3. - I'd like to keep `Utilities` and `Tooling` separate. Would you accept that change?

**Comment by robdimsdale** [comment]
I have updated the [upstream](https://github.com/cloudfoundry/community/compare/main...ryanmoran:community:buildpacks-subteams) but the commits aren't showing here. I don't know why this is the case, and I can't seem to fix it.

I'm going to open a new PR from my fork (rather than Ryan's). Can someone close this PR in favor of the new one please?

**Comment by Gerg** [comment]
Closing in favor of https://github.com/cloudfoundry/community/pull/586

**Pull request closed without merging**
